core/templates/dev/head/services/navigation-bars-backend-api.service.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
@bansalnitish, have we started migrating services to Angular? If yes, then should we prefer the new **independent** services in Angular? (Just to avoid re-work)

Reply:
(Adding [link](https://github.com/oppia/oppia/pull/8155#pullrequestreview-336702559) to @nishantwrp's similar comment.)

Reply:
@DubeySandeep I'm not familiar with Angular or AngularJS (or the difference between the two), was just looking at similar files in the codebase to figure out how to do it. Can you point me to an example of a service that is done the right way?

Reply:
Hey @cdrebin Can you view this [sheet](https://docs.google.com/spreadsheets/d/1L9Udn-XT6Lk1qaTBUySTw1AnhvQMR-30Qry4rfd-Ovg/edit#gid=0)? The ones markes as migrated are all in Angular